### ISSUE TITLE
Using parameters when getting values?

### ISSUE THREAD

**Issue by GollyJer** [keajs/kea#138]
Hi! Easy-Peasy seems to have been abandoned. And while I love it, it's hard to rely on going forward.

Kea is at the top of my list as a replacement but please bear with me as I'm starting fresh after reading through the docs.

I'm curious how I might do something like [Computed with parameters](https://easy-peasy.vercel.app/docs/api/computed.html) from easy-peasy?

sample code... `collectedAssets` is a computed property.
```javascript
const ownedAssets = useAppState((state) => state.collectedAssets(collection.id));
```

Basically, `computed` is similar to Kea's `selector` (using other pieces of state to return a value) but it allows for passing in parameters.

How would this be accomplished with Kea?  Thanks!

**Comment by mariusandra** [comment]
Hey @GollyJer , although it's nowhere in the docs, nothing prohibits you from returning a function from a [selector](https://kea.js.org/docs/guide/concepts#selectors) instead of just a value. So if you want a drop-in replacement, the code would be something like this:

```js
const logic = kea({
  selectors: {
    collectedAssets: [
      (s) => [s.assets], // or wherever you get them from
      (assets) => (id) => assets.find(a => a.id === id)
    ]
  }
})

function MyComponent({ id }) {
  const { collectedAssets } = useValues(logic)
  const ownedAssets = collectedAssets(id)
  // ...
}
```

That said, depending on your code, it might be better to give the `id` directly to kea [as a `key`](https://kea.js.org/docs/guide/additional#keyed-logic) and mount your logic per component, eliminating the need for such a lookup... and further disconnecting your [data layer](https://kea.js.org/blog/2021/05/14/data-first-frontend-revolution) from from React:

```js
const assetsLogic = kea({ reducers: { assets: [null] } }) // get them somehow

const collectionLogic = kea({
  key: (props) => props.id,
  selectors: {
    ownedAssets: [
      () => [assetsLogic.selectors.assets, (state, props) => props.id],
      (assets, id) => assets.find(a => a.id === id)
    ]
  }
})

function MyComponent({ id }) {
  const { ownedAssets } = useValues(collectionLogic({ id }))
  // ...
}
```

As for easy-peasy, [I don't think it has been abandoned](https://github.com/ctrlplusb/easy-peasy/issues/679#issuecomment-903530090). Maintaining a project can be quite a lot of work, and it's easy for that work to take the backseat when life throws more important things at you. Seems like that's what happened now.

That said, Kea is not going away either. I'm working with it daily at [PostHog](https://posthog.com) in the experience team, and will be releasing various updates and new plugins in the near future (e.g. [kea-forms](https://github.com/keajs/kea-forms)).

**Comment by GollyJer** [comment]
Awesome. Thanks @mariusandra!
